Demos/benchmarks/setup.py

```diff
@@ -9,5 +9,5 @@
   name = 'benchmarks',
   ext_modules = cythonize("*.py", language_level=3, annotate=True,
                           compiler_directives=directives,
-                          exclude="setup.py"),
+                          exclude=["setup.py","chaos.py"]),
```

Comment:
updating to v0.24 fixes chaos.py for 2.7 and 3.5, and embed and freeze for 3.5. I've updated the makefile and readme